packages/lib/src/interactions/InteractionsProvider.tsx

```diff
@@ -79,10 +79,11 @@ function InteractionsProvider(props: { children: ReactNode }) {
       }
 
       // If conflicting interactions, the one with the most modifier keys take precedence
-      const [maxKeyInteraction] = matchingInteractions
-        .sort((a, b) => a.modifierKeys.length - b.modifierKeys.length)
-        .reverse();
+      matchingInteractions.sort(
+        (a, b) => b.modifierKeys.length - a.modifierKeys.length
+      );
```

Comment:
This is https://github.com/cartant/eslint-plugin-etc/blob/main/docs/rules/no-assign-mutated-array.md. Here, it's now clear that the array is sorted in place. Note that I also remove the need for `reverse()` by tweaking the comparator function.